lib/membrane_opus/parser.ex

```diff
@@ -116,11 +130,27 @@ defmodule Membrane.Opus.Parser do
           channels :: 0..2
         ) ::
           {:ok, remaining_buffer :: binary, pts :: Membrane.Time.t(), packets :: [Buffer.t()],
-           channels :: 0..2}
+           channels :: 0..2, generate_best_effort_timestamps? :: boolean()}
```

Comment:
You have put `generate_best_effort_timestamps?` in the function return type, while it should be added to the function args